### DISCUSSION TITLE
Just bought it - love it - want more!

### DISCUSSION THREAD

**Discussion by ethereal-engineer** [Difegue/Stylophone#81] [General]
I have more than one mpd server on site and I would LOVE the ability to register and use multiple servers. That's about it. Aside from this one feature lacking, this app leaves the rest in the dust!

**Comment by Difegue** [comment]
Thanks, glad you enjoy it! I'm keeping track of interest in multi-server support in #22.  
It'd be mostly UI work at this point I think, so it might make the cut for a future release. 🙇
